tensorflow/python/ops/image_ops_impl.py

```diff
@@ -2162,7 +2162,7 @@ def is_jpeg(contents, name=None):
     name: A name for the operation (optional)
 
   Returns:
-     A scalar boolean tensor indicating if 'contents' may be a JPEG image.
+     A scalar boolean tensor indicating if 'contents' maybe a JPEG image.
```

Comment:
This should be "may be". Similar in other places downwards

Reply:
@mihaimaruseac made the required changes